.github/workflows/copilot-test-e2e.yml

```diff
@@ -3,17 +3,19 @@ on:
   workflow_dispatch:
   merge_group:
     types: [checks_requested]
+  pull_request_review:
+    types: [submitted]
 
 permissions:
   contents: read
 
 jobs:
   e2e:
+    if: github.event.review.state == 'APPROVED' || github.event_name == 'workflow_dispatch'
     defaults:
       run:
         working-directory: webapp
     runs-on: ubuntu-latest
-    timeout-minutes: 30
```

Comment:
I'm guessing you'll put a timeout back once you know how long it takes to run these tests?

Also, I find it suspicious that it wouldn't complete in 30 minutes!

Reply:
Or was that just moved to the step?

Reply:
Yes